quarkus-jooq-pro/deployment/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>io.quarkiverse.jooq.pro</groupId>
         <artifactId>quarkus-jooq-parent</artifactId>
-        <version>0.2.2-SNAPSHOT</version>
```

Comment:
Ah this should have been `999-SNAPSHOT`. And we probably need to tweak the release process as there's something odd going on.

Reply:
Yes. What's going on is, I guess, that some version setting tool (mvn versions:set? or mvn release:*?) are not run with the `-Pjooq-pro`, and therefore these submodules were being skipped since 0.2.2.

That's probably  because the build does not have the JOOQ pro licensed libraries available. But that can likely be overcame by just using JOOQ non-pro libs, copied under the other groupId.

Reply:
:ballot_box_with_check:

Reply:
I have experimented with disabling `versions:set` if `-Pjooq-pro` is not set. But it seems that versions plugin is special and does not react to an override of the `<phase>`, and there is no `<skip>` in it's config.  

See https://github.com/OndraZizka/x-quarkus-jooq/pull/1